### PR TITLE
Adds the medical labcoat as the default MD suit + Adds it to the medical wardrobe

### PR DESCRIPTION
**Pull request by nikothedude**
## About The Pull Request
Title. 

Also adds a new module, job_loadout_overrides, which is for overridding the default loadouts of jobs.

## How This Contributes To The Skyrat Roleplay Experience
The sprite is better than the original labcoat. Also, it's the MEDICAL labcoat, doctors shouldn't be stuck with the generic `labcoat`.
image

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>

image
image

</details>

## Changelog

:cl:
add: The medical labcoat has been added to the medidrobe and loadout.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/job_loadout_overrides/medical_doctor.dm
Comment (Zonespace27) [inline modular_skyrat/modules/job_loadout_overrides/medical_doctor.dm]: please put this in master_files or something; it doesn't need a whole module

Comment (nikothedude) [reply modular_skyrat/modules/job_loadout_overrides/medical_doctor.dm]: tomorrow

**Comment by theselfish** [comment]
I dislike this.
But, honestly, I dislike the stripes on all of the labcoats.
At the very least have the original one there.

Blue stripes harkins to the Genetics department.
It'd be like doing something like making Security blue, or something.

**Comment by nikothedude** [comment]
> I dislike this. But, honestly, I dislike the stripes on all of the labcoats. At the very least have the original one there.
> 
> Blue stripes harkins to the Genetics department. It'd be like doing something like making Security blue, or something.

1. The default labcoat is still in loadout + medidrobe
2. Medical's color scheme is literally white and blue. Just 'cause genetics has it doesnt mean we shouldnt use it

**Comment by ghost** [comment]
> I dislike this. But, honestly, I dislike the stripes on all of the labcoats. At the very least have the original one there.
> 
> Blue stripes harkins to the Genetics department. It'd be like doing something like making Security blue, or something.

Genetics actually has this labcoat by default anyways, yes
HONESTLY, this does make me curious about using the color used for medical scrubs instead of the light blue, but having a whole new labcoat would be even more redundant

Does **any** department use the plain one any more besides medical? Science has a purple marked one......

**Comment by theselfish** [comment]
> > I dislike this. But, honestly, I dislike the stripes on all of the labcoats. At the very least have the original one there.
> > Blue stripes harkins to the Genetics department. It'd be like doing something like making Security blue, or something.
> 
> Genetics actually has this labcoat by default anyways, yes HONESTLY, this does make me curious about using the color used for medical scrubs instead of the light blue, but having a whole new labcoat would be even more redundant
> 
> Does **any** department use the plain one any more besides medical? Science has a purple marked one......

No. Science has Purple, Genetics has blue, Viro has green, the CMO has an entirely blue one, the RD has an odd one and Robotics gets a black/red one if memory serves correctly.

The white coat is the Medical one.

**Comment by Paxilmaniac** [comment]
This should be a loadout option locked to doctor instead I think, rather than replacing the labcoat for everyone medical as the default.

**Comment by nikothedude** [comment]
I'm personally indifferent on it either being in loadout or default, so I'll let a maintainer decide.

**Comment by Zergspower** [comment]
I dont see an issue with a medical labcoat being in the medical vendor? My prominant issue with loadouts is you're limited on points for the job specific ones, if those were moved to 0 cost then sure but i dont wanna change out my loadout everytime i play a different job.

**Review by tf-4 — CHANGES_REQUESTED** [review]
Agree with pax

**Comment by nikothedude** [comment]
Done

**Review by tf-4 — APPROVED** [review]

**Pull request merged**
